src/plugins/workspace/public/components/delete_workspace_modal/delete_workspace_modal.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFieldText,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+
+interface DeleteWorkspaceModalProps {
+  selectedItems: string[];
+  visible: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+// eslint-disable-next-line import/no-default-export
+export default function DeleteWorkspaceModal(props: DeleteWorkspaceModalProps) {
+  const [value, setValue] = useState('');
+  const { onClose, onConfirm, visible, selectedItems } = props;
```

Comment:
I think it's better to control the visibility of DeleteWorkspaceModal at `workspace_overview.tsx`, it should be like 
```typescript
{deleteWorkspaceModalVisible && <DeleteWorkspaceModal />}
```